nttt/tidyup.py

```diff
@@ -39,7 +39,8 @@ def increase_indent(self, flow=False, indentless=False):
     return '---\n' + yaml.dump(parsed_md,
                                Dumper=IndentedDumper,
                                allow_unicode=True,
-                               sort_keys=False)
+                               sort_keys=False,
+                               width=160)
```

Comment:
why not 1000000? We want to avoid those line breaks completely.
Ideally it would be best to prevent the dumper from introducing those breaks at all.
Or valid YAML syntax for them.

Reply:
The dumper we use always imposes a maximum line length. The default (80) is not enough, but given the nature of the meta.yml file, something like 160, or maybe 1000 to be really safe, should be enough. 1000000 sounds a bit over the top, but doesn't do any harm either.